source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -121,6 +121,7 @@ - (void)viewDidLoad
     ADCResolver *resolver = [[ADCResolver alloc] init];
     [_resolvers setResourceResolver:resolver scheme:@"http"];
     [_resolvers setResourceResolver:resolver scheme:@"https"];
+    [_resolvers setResourceResolver:resolver scheme:@"data"];
```

Comment:
Do we have unit tests for this code path today that can run as part of CI (so automated unittests-based verification rather than manual functional/end-end)?

Reply:
No, we don't have it. I will add it.

---
In reply to: [387934866](https://github.com/microsoft/AdaptiveCards/pull/3817#discussion_r387934866) [](ancestors = 387934866)